web/static/height/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AutoKitteh - Height</title>
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/static/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/static/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/static/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/static/site.webmanifest" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta name="robots" content="noindex, nofollow" />
+    <script src="form.js" defer></script>
+  </head>
+
+  <body>
+    <a href="/" target="_self">
+      <img class="banner" src="/static/banner.svg" alt="AutoKitteh" />
+    </a>
+
+    <table class="title">
+      <td>
+        <img class="logo" src="/static/images/height.svg" alt="Logo" />
+      </td>
+      <td><h1>Height - Initialize Connection</h1></td>
+    </table>
+
+    <form method="post" action="/height/save">
+      <h2>Query Params From the URL</h2>
+
+      <div class="form-group">
+        <label for="cid">Connection ID</label>
+        <input type="text" name="cid" id="cid" readonly />
+      </div>
+
+      <div class="form-group">
+        <label for="origin">Request Origin</label>
+        <input type="text" name="origin" id="origin" readonly />
+      </div>
+
+      <h2>Authentication Type</h2>
+
+      <div class="form-group">
+        <select name="auth_type" id="authType">
+          <option value="oauthDefault">Default OAuth 2.0</option>
+          <option value="oauthPrivate">Private OAuth 2.0</option>
+        </select>
+      </div>
+
+      <div id="oauthSection">
+        <h2>OAuth 2.0</h2>
+      </div>
+
```

Comment:
Unlike Linear, there's no input which applies to both default & private OAuth apps, so this section isn't needed.

```suggestion
```